packages/pricing/integration-tests/__tests__/services/price-list-rule/index.spec.ts

```diff
@@ -0,0 +1,243 @@
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+
+import { PriceListRuleRepository } from "@repositories"
+import { PriceListRuleService } from "@services"
+
+import { createPriceLists } from "../../../__fixtures__/price-list"
+import { createPriceListRules } from "../../../__fixtures__/price-list-rules"
+import { createRuleTypes } from "../../../__fixtures__/rule-type"
+import { MikroOrmWrapper } from "../../../utils"
+
+jest.setTimeout(30000)
+
+describe("PriceListRule Service", () => {
+  let service: PriceListRuleService
+  let testManager: SqlEntityManager
+  let repositoryManager: SqlEntityManager
+
+  beforeEach(async () => {
+    await MikroOrmWrapper.setupDatabase()
+    repositoryManager = await MikroOrmWrapper.forkManager()
+
+    const priceListRuleRepository = new PriceListRuleRepository({
+      manager: repositoryManager,
+    })
+
+    service = new PriceListRuleService({
+      priceListRuleRepository,
+    })
+
+    testManager = await MikroOrmWrapper.forkManager()
+    await createRuleTypes(testManager)
+    await createPriceLists(testManager)
+    await createPriceListRules(testManager)
+  })
+
+  afterEach(async () => {
+    await MikroOrmWrapper.clearDatabase()
+  })
+
+  describe("list", () => {
+    it("list priceListRules", async () => {
```

Comment:
suggestion: Don't forget that the tests should be readable as a sentence such as `should return a list of priceListRules`. The more they look like sentence the easier it will be to read the description in general for outside users as well 👍 I think we agreed on that sometimes ago